line-bot-model/src/main/java/com/linecorp/bot/model/Broadcast.java

```diff
@@ -46,6 +47,18 @@
      */
     private final boolean notificationDisabled;
 
+    public Broadcast(Message messages) {
```

Comment:
I think what we should do is adding `@Builder`. could you wait adding those in this PR?

Reply:
Is adding `@Builder` done by issue-254?

Reply:
Yes. How about just adding `public Broadcast(Message messages, boolean notificationDisabled) {`?